### PR TITLE
[mfgui-url] show mfgui url in console

### PR DESCRIPTION
**Pull request by jackie-ob**

### CODE REVIEW

**Comment by savingoyal** [comment]
Can we file an issue for a follow up PR that prints the UI URL for `step-functions trigger` and `argo-workflows trigger`?

File: metaflow/runtime.py [hunk 1]
```diff
@@ -198,8 +198,19 @@ def persist_constants(self, task_id=None):
...
 
     def execute(self):
-
-        self._logger("Workflow starting (run-id %s):" % self._run_id, system_msg=True)
+        if MFGUI_URL:
+            self._logger(
+                "Workflow starting (run-id %s), see it in the UI at %s"
+                % (
+                    self._run_id,
+                    os.path.join(MFGUI_URL, self._flow.name, self._run_id),
```
Comment (shrinandj) [inline metaflow/runtime.py:206]: Should this be `urlib.parse.urljoin` (or a similar function) that does url joins? os.path.join would result in the wrong url on non-unix envs. Not sure how if Metaflow is used on non-envs though.

Comment (romain-intel) [reply metaflow/runtime.py:206]: the current assumption is mf only works on linux like systems. Your point remains valid but I think the assumption is pretty pervasive.

**Review by romain-intel — APPROVED** [review]
Apart from a minor comment on naming, lgtm.

**Comment by jackie-ob** [comment]
Noted.  We will switch to `urlib.parse.urljoin` and `METAFLOW_UI_URL`

**Review by savingoyal — DISMISSED** [review]
good to go after pending changes are shipped!

File: metaflow/runtime.py (hunk 1 above)
Comment (jackie-ob) [reply metaflow/runtime.py]: Note: we switched to just using manual string construction.  `urljoin` does not do what we want (e.g. `urljoin("http://A/B", "C") == "http://A/C"`.

**Pull request merged**